### DISCUSSION TITLE
Conditionally showing a form item based on another form item's input value

### DISCUSSION THREAD

**Discussion by gregsdennis** [community/community#64437] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

[Docs reference](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema)

I have an issue template form YAML file declaring a dropdown selector.  Below the dropdown, I'd like to conditionally show some text based on the item that is selected.

Is this possible?

**Comment by WilliamHCarter** [comment]
Unfortunately, I don't think its possible to conditionally show text in this manner. With that being the case, it's probably best to just display the text under each option, as annoying as that may feel.

**Reply by gregsdennis** [reply]
> display the text under each option

Do you mean show the text under the item _inside the dropdown_?  I'm not sure that's possible.  Also, I'm planning on putting links (e.g. to documentation) in the text.  Users wouldn't be able to click on a link if it's in a dropdown.
